extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcConfig.java

```diff
@@ -58,6 +59,12 @@
     @ConfigItem
     Roles roles;
 
+    /**
+     * Configuration how to validate the token claims.
+     */
+    @ConfigItem
+    Claims claims;
```

Comment:
I'm not sure about `quarkus.oidc.claims` config property. It may clash with the `claims` request parameter as defined by https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter.

Maybe, we could use `quarkus.oidc.token.issuer|audience` ?

Reply:
@pedroigor Good point, please believe me a `token` qualifier was something I thought for a sec or two about as well :-), because we may want to actually have an option to have both `id token` and `access token` validated separately, right now this validation will affect either the `id token` or `access token` depending on the application type. But in time we may qualify further and introduce an `idtoken` group if needed.
Recall though we already have an `Authentication` group which has `scopes` but will also host more properties related to the redirect, but in any case, it is good to avoid the confusion, so I'll rename shortly
Thanks

Reply:
@pedroigor done